### PR TITLE
Fix/nex 1989/feature flag undefined variable

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
https://oat-sa.atlassian.net/browse/NEX-1989

I updated `.ENV` variable in order to check if key exist. 

README updated with small instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,42 @@
 tao-core
 ========
 
-[Task Queue Doc](models/classes/taskQueue/README.md)
\ No newline at end of file
+[Task Queue Doc](models/classes/taskQueue/README.md)
+
+# Feature flag
+
+You can now define feature flag in `.ENV` global variables.
```
Comment (gabrielfs7) [inline README.md:8]:
```suggestion
You can now define feature flag as environment variables. 
```

Cause `.ENV` is a bit misleading. People might thing you need a .env file or something.

Comment (gabrielfs7) [reply README.md:8]: It is also important to state that the standard is to have the prefix `FEATURE_FLAG_XXXXX` at some part. Wdyt?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
+    ]
+]
+```
+
+This configuration will display `formField_01` when `FEATURE_FLAG_NAME` is enabled
+
+### SectionVisibilityFilterInterface
+
+`tao/SectionVisibilityFilter` is responsible for listing sections that can be disabled from user. In order to add more 
+sections that has to be disabled/enabled based on feature flag.
```
Comment (gabrielfs7) [inline README.md:32]:
```suggestion
sections that have to be disabled/enabled based on feature flag. 
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,42 @@
...
+`tao/SectionVisibilityFilter` is responsible for listing sections that can be disabled from user. In order to add more 
+sections that has to be disabled/enabled based on feature flag. 
+
+```
+'featureFlagSections' => [
+    'sectionName' => [
+        `FETURE_FLAG_01`
+    ]
+]
+```
```
Comment (gabrielfs7) [inline README.md:40]: Maybe use the php configuration file content to be more precise? Wdyt?

File: README.md [hunk 4]
```diff
@@ -1,4 +1,42 @@
...
+
+To hide/show form fields you can extend `AbstractFeatureFlagFormPropertyMapper` with your configurable service that has
+`OPTION_FEATURE_FLAG_FORM_FIELDS` mapped to list of fields that you want to hide/show. 
+
+```
+FeatureFlagFormPropertyMapper::OPTION_FEATURE_FLAG_FORM_FIELDS => [
+    formField_01 => [
+        FEATURE_FLAG_NAME
+    ]
+]
```
Comment (gabrielfs7) [inline README.md:24]: Maybe use the php configuration file content to be more precise? Wdyt?

**Comment by gabrielfs7** [comment]
Needs to resolve conflicts for manifest

**Review by gabrielfs7 — APPROVED** [review]

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: models/classes/featureFlag/FeatureFlagChecker.php [hunk 5]
```diff
@@ -28,6 +28,10 @@ class FeatureFlagChecker extends ConfigurableService implements FeatureFlagCheck
 {
     public function isEnabled(string $feature): bool
     {
+        if (!isset($_ENV[$feature])) {
```
Comment (mike-ionut-mihai-sandu-tao) [inline models/classes/featureFlag/FeatureFlagChecker.php:31]: do we need to update the test?

Comment (bartlomiejmarszal) [reply models/classes/featureFlag/FeatureFlagChecker.php:31]: I ave all test required to check this change:
vriable exist, not exist, is disabled

**Pull request merged**